cluster-migration/src/main/java/com/wipro/ats/bdre/clustermigration/DestTableLoad.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *  
+ */
+
+package com.wipro.ats.bdre.clustermigration;
+
+import com.wipro.ats.bdre.BaseStructure;
+import org.apache.commons.cli.CommandLine;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.apache.log4j.Logger;
+
+/**
+ * Created by cloudera on 4/7/16.
+ */
+public class DestTableLoad extends BaseStructure {
+    private static final Logger LOGGER = Logger.getLogger(DestTableLoad.class);
+    private static final String[][] PARAMS_STRUCTURE = {
+            {"src", "source-path", " Source path"},
+            {"dest", "dest-path", " Destination path"},
+            {"destFs", "dest-fs", " Destination file system"}
+    };
+
+    public void execute(String[] params) throws Exception{
+
+        CommandLine commandLine = getCommandLine(params, PARAMS_STRUCTURE);
+        String src = commandLine.getOptionValue("source-path");
+        String dest = commandLine.getOptionValue("dest-path");
+        String destFs=commandLine.getOptionValue("dest-fs");
+
+        Configuration config = new Configuration();
+        config.set("fs.defaultFS",destFs);
+        FileSystem hdfs = FileSystem.get(config);
+        Path srcPath = new Path(src);
+        Path destPath = new Path(dest);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "destPath" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)